package.json

```diff
@@ -29,15 +29,15 @@
     "ember-cli-htmlbars-inline-precompile": "^0.3.1",
     "ember-cli-inject-live-reload": "^1.4.0",
     "ember-cli-jshint": "^1.0.0",
-    "ember-cli-qunit": "^1.4.0",
+    "ember-cli-qunit": "^3.1.1",
     "ember-cli-release": "0.2.8",
     "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "^1.2.0",
     "ember-disable-prototype-extensions": "^1.1.0",
     "ember-export-application-global": "^1.0.5",
     "ember-load-initializers": "^0.5.1",
     "ember-resolver": "^2.0.3",
-    "ember-suave": "2.0.1",
+    "ember-suave": "4.0.1",
```

Comment:
Had to update because tests were failing after bumping ember-cli-qunit

Reply:
This probably needs to be removed in #60 because [ember-suave](https://github.com/DavyJonesLocker/ember-suave) is no longer maintained (it's based on JSCS)